packages/canary-web-components/src/components/ic-tree-item/ic-tree-item.css

```diff
@@ -130,11 +138,23 @@
     fill: currentcolor;
   }
 
+  :host .tree-item-content:hover,
+  ::slotted([slot="router-item"]:hover),
+  :host .tree-item-content:active,
+  ::slotted([slot="router-item"]:active),
+  :host(.ic-tree-item-selected) .tree-item-content,
+  :host(.ic-tree-item-selected) ::slotted([slot="router-item"]) {
+    background-color: none !important;
+  }
+
   :host(.ic-tree-item-disabled) .tree-item-content,
   :host(.ic-tree-item-disabled) .arrow-dropdown,
   :host(.ic-tree-item-disabled) ::slotted([slot="icon"]),
   :host(.ic-tree-item-disabled) ::slotted([slot="router-item"]) {
     color: GrayText !important;
+
+    --ic-typography-color: GrayText !important;
```

Comment:
Will this be replaced with a design token in future work?

Reply:
not these are they are system High Contrast colors that we should be using